packages/api-clients/open-api/m2mGatewayApi.yml

```diff
@@ -10355,6 +10468,89 @@ components:
       required:
         - version
         - answers
+    EServiceRiskAnalysisSeed:
+      type: object
+      additionalProperties: false
+      required:
+        - name
+        - riskAnalysisForm
+      properties:
+        name:
+          type: string
+        riskAnalysisForm:
+          $ref: "#/components/schemas/RiskAnalysisFormSeed"
+    EServiceRiskAnalysis:
+      type: object
+      additionalProperties: false
+      required:
+        - id
+        - name
+        - riskAnalysisForm
+        - createdAt
+      properties:
+        id:
+          type: string
+          format: uuid
+        name:
+          type: string
+        riskAnalysisForm:
+          $ref: "#/components/schemas/RiskAnalysisForm"
+        createdAt:
+          type: string
+          format: date-time
+    RiskAnalysisForm:
```

Comment:
the model should be similar to the seed (as a reference, see `RiskAnalysisForm` in the `purpose-process` api)

Reply:
Done, tracked here a future task to align catalog process to expose the same model directly: [PIN-7452](https://pagopa.atlassian.net/browse/PIN-7452)